provider/resource_workflow_task_update_opsgenie_incident.go

```diff
@@ -73,7 +73,7 @@ func resourceWorkflowTaskUpdateOpsgenieIncident() *schema.Resource {
 							Description: ". Value must be one of `resolve`, `open`, `auto`.",
 							Type: schema.TypeString,
 							Optional: true,
-							Default: "resolve",
+							Default: nil,
```

Comment:
This doesn't match the description

Description: ". Value must be one of `resolve`, `open`, `auto`.",

Reply:
@kwent  we chatted about this yesterday? Since they aren't required values I still think the doc makes sense. It also matches behaviour of web ui (same for comment below).

Reply:
ha gotcha. Can remove this weird extra dot at the beginning of desc

Reply:
Changes will be on Rootly Repo.